sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -125,6 +140,10 @@ def create_rsa_key(self, name, **kwargs):
         :paramtype tags: dict[str, str]
         :keyword ~datetime.datetime not_before: Not before date of the key in UTC
         :keyword ~datetime.datetime expires_on: Expiry date of the key in UTC
+        :keyword bool exportable: Whether the private key can be exported.
```

Comment:
```suggestion
        :keyword bool exportable: Whether the key can be exported.
```
There's no public/private separation here, it's just a single key

Reply:
Hm, would a more accurate description be whether the private component of the key can be exported? This wording was based on [JS's](https://github.com/Azure/azure-sdk-for-js/pull/16150/files#diff-bd87b9a2319e7bf24d7284d987cadf4a3d1e6fceb32aa96ef7ffc21f0452a692R219) and the generated model's property description: "Indicates if the private key can be exported."

Reply:
It's a subtlety for oct (AES) since it's a symmetric key, but I agree that Charles's suggestion is more accurate. When declared on a model - or on a class like .NET and Java have - we can only have a single comment and "private key" is more accurate for all 3 supported algorithms than "key" would be misleading.